site/src/pages/TemplateVersionEditorPage/TemplateVersionEditor.tsx

```diff
@@ -162,90 +157,196 @@ export const TemplateVersionEditor: FC<TemplateVersionEditorProps> = ({
       ["running", "pending"].includes(previousVersion.current.job.status) &&
       templateVersion.job.status === "succeeded"
     ) {
-      setSelectedTab("resources");
       setDirty(false);
     }
     previousVersion.current = templateVersion;
   }, [templateVersion]);
 
-  const hasIcon = template.icon && template.icon !== "";
-  const showBuildLogs = Boolean(buildLogs);
   const editorValue = getFileContent(activePath ?? "", fileTree) as string;
 
+  // Auto scroll
+  const buildLogsRef = useRef<HTMLDivElement>(null);
   useEffect(() => {
-    window.dispatchEvent(new Event("resize"));
-  }, [showBuildLogs]);
+    if (buildLogsRef.current) {
+      buildLogsRef.current.scrollTop = buildLogsRef.current.scrollHeight;
+    }
+  }, [buildLogs]);
```

Comment:
1 - No need to use `useLayoutEffect` for now since the panel only shows when the build happens. Even without anything, the scroll would behave the same. At least on my tests, I didn't see anything strange.
2 - No problem with the scroll happening when it is undefined, there is no side effect on that.
3 - I think it is a good one but I see this as an "upgrade" of the feature. If we see users asking for that, we could definitely implement it.